docs/QSG/setup_suse.md

```diff
@@ -85,31 +81,25 @@ daos-server RPM.
 
 		pdsh -w $ALL_NODES 'sudo zypper ar https://packages.daos.io/v2.4/Leap15/packages/x86_64/ daos_packages'
 
+		pdsh -w $ALL_NODES 'sudo zypper ar https://packages.daos.io/v2.4/Leap15/packages/x86_64/ daos_packages'
```

Comment:
This kind of stinks. @johannlombardi: Can't we make a `current` or `latest` link so that we don't need to keep updating documentation?